packages/rum-recorder/test/utils.ts

```diff
@@ -69,6 +69,8 @@ export function collectAsyncCalls<F extends jasmine.Func>(spy: jasmine.Spy<F>) {
       spy.and.callFake((() => {
         fail('Unexpected extra call')
       }) as F)
+      // make sure the "setTimeout" is not mocked
+      jasmine.clock().uninstall()
```

Comment:
In recorder.spec.ts, I now wait for the segment to expire instead of notifying a BEFORE_UNLOAD lifecycle event. Thus, I needed to mock the clock with jasmine.

But, the code tested is still asynchronous because it uses WebWorker communications, so I still need to use `collectAsyncCalls`, and  use an actual `setTimeout` to wait for WebWorker messages to be sent.

So I could ensure `setTimeout` to not be mocked and throw an error, but I'd still need to unmock it in all my tests before using `expectNoExtraAsyncCalls`. I thought that removing the mock here makes things a bit less verbose and a bit more fool proof.

Reply:
Anyway, I'll just revert this commit, since it is now unnecessary with the VIEW_ENDED lifecycle event :)